src/runtime/opencl.cpp

```diff
@@ -99,6 +103,15 @@ WEAK char build_options[1024];
 WEAK ScopedSpinLock::AtomicFlag build_options_lock = 0;
 WEAK bool build_options_initialized = false;
 
+template<typename... Args>
+halide_error_code_t error_opencl(void *user_context, cl_int cl_error, const Args &...args) {
+    if (cl_error == CL_SUCCESS) {
+        return halide_error_code_success;
+    }
+    error(user_context).append(get_opencl_error_name(cl_error), args...);
```

Comment:
Similar change should happen here to the cuda change:

```
    error(user_context).append("OpenCL error: ", get_opencl_error_name(cl_error), " ", args...);
```

Reply:
Fixed